woocommerce/woocommerce-admin
# Onboarding: Update screen order and remove Jetpack connection in profiler

Fixes (part of) #3709 

* Moves the plugin install screens to the end of the profiler
* Removes the Jetpack connection from the profiler
* Removes the cart modal from the profiler
* Removes usage modal from plugins screen and enforces it on store details step
* Fixes some existing event props

@pmcpinto @jameskoster Some of the wording on the "Benefits" step (previously the start step) is weird since it's at the end.  `Start setting up your WooCommerce store` - not sure if we want to do any quick changes since the new design changes will fix this anyway.

### Detailed test instructions:

1. Delete the onboarding data `woocommerce_onboarding_profile`
1. Try deactivating/deleting a combination of Jetpack and WCS plugins and also try with Jetpack connected/disconnected.
1. Enable the profiler.
1. Make sure the screen order is correct. (Benefits and Plugins screen moved to the end).
1. Make sure usage modal is enabled during store details step.
1. Walk through all the steps until the Benefits page (previously "start" page).
1. Make sure both "Get Started" and "skip" options work as expected.
1. Make sure plugins screen installs plugins and completes profiler after activation.
1. Make sure no cart modal is shown.
